Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -967,6 +967,7 @@ class RenderStyle {
     const ScrollSnapAlign& scrollSnapAlign() const;
     ScrollSnapStop scrollSnapStop() const;
 
+    inline const std::optional<ScrollbarColor> scrollbarColor() const;
```

Comment:
Do we need the `inline` here 🤔 ?

Reply:
If your question is related to whether it should be in RenderStyleInlines or RenderStyle then I'm also not sure on the decision that goes into choosing between them. It was rather arbitrary tbh. Any guidance on this would be great.

Reply:
I think @darinadler did changes around RenderStyle to speed-up builds and had separate 'RenderStyleInlines' as a result. I would personally just go back to particular commit and read commit message to understand the rationale, hope it answers. Just wanted to help how I can.

Reply:
We use 'inline' or 'constexpr' when there is an inline function implementation in any other header. It guarantees we get a warning if we forget to include that header, even in builds with inlining disabled, like debug builds.

We don't need to use 'inline' when the function body is in the same header. No harm, but we typically omit it.

@jernoble started doing this and I followed suit.

Reply:
I think it's fine like this.
In general in the WebKit codebase, we keep bare getter/setter in headers and anything else in the implementation file.